ui/src/components/Sidebar/MobileSidebar.tsx

```diff
@@ -29,12 +33,17 @@ function GroupsTab(props: { isInactive: boolean; isDarkMode: boolean }) {
     <DoubleClickableNavTab
       onSingleClick={onSingleClick}
       onDoubleClick={() => navigate('/')}
-      linkClass="basis-1/5"
     >
       <HomeIconMobileNav
         isInactive={props.isInactive}
         isDarkMode={props.isDarkMode}
-        className="mb-0.5 h-6 w-6"
+        className="h-6 w-6"
+      />
+      <div
+        className={cn(
+          'mt-[2px] h-1.5 w-1.5 rounded-full',
```

Comment:
<img width="424" alt="image" src="https://github.com/tloncorp/landscape-apps/assets/5466421/68d25e0f-d273-4ad4-8556-6bf22a44545e">

so our icons in the FE are sized at 24px with no frame, but in the nav mocks here their frame is 32px with the dot butt up against the frame.

<img width="175" alt="image" src="https://github.com/tloncorp/landscape-apps/assets/5466421/b912815a-a348-4d2f-9fe2-1a3a75295c3c">

I think that distance is 4px between icon and bottom of frame so in the outlines here is top margin at 4px which I think is closer to the mocks.